openpype/hosts/houdini/api/lib.py

```diff
@@ -138,6 +138,9 @@ def get_output_parameter(node):
             return node.parm("ar_ass_file")
     elif node_type == "Redshift_Proxy_Output":
         return node.parm("RS_archive_file")
+    elif node_type == "ifd":
```

Comment:
Is this meant for this PR? Just wondering.

Reply:
yes, as this PR is also adding ifd implementation which is also needed for farm caching.....